docs/azure-sql-db.md

```diff
@@ -0,0 +1,163 @@
+  ```
+
+  If you can not find the service name, please use the following command to make the plans public.
+
+  ```
+  cf enable-service-access azuresqldb
+  ```
+
+2. Create a service instance
+
+  Configuration parameters are supported with the provision request. These parameters are passed in a valid JSON object containing configuration parameters, provided either in-line or in a file.
+
+  ```
+  cf create-service azuresqldb $service_plan $service_instance_name -c $path_to_parameters
+  ```
+
+  Supported configuration parameters:
+
+  ```
+{
+    "resourceGroup": "<resource-group>",
+    "sqlServerName": "<sql-server-name>",
+    "sqlServerCreateIfNotExist": true | false,
+    "sqlServerParameters": {
+        "location": "<azure-region-name>",
+        "properties": {
+            "administratorLogin": "<sql-server-admin-name>",
+            "administratorLoginPassword": "<sql-server-admin-password>"
+        }
+    },
+    "sqldbName": "<sql-database-name>",
+    "sqldbParameters": {
+        "location": "<azure-region-name>",
+        "properties": {
+            "collation": "SQL_Latin1_General_CP1_CI_AS | <or-other-valid-sqldb-collation>"
+        }
+    }
+}
+  ```
+
+  For example:
+
+  ```
+  cf create-service azuresqldb basic mysqldb -c /tmp/config.json
+  ```
+
+  ```
+  {
+    "resourceGroup": "sqldbResourceGroup",
+    "sqlServerName": "sqlServerA",
+    "sqlServerCreateIfNotExist": true,
+    "sqlServerParameters": {
+        "location": "westus",
+        "properties": {
+            "administratorLogin": "ulrich",
+            "administratorLoginPassword": "u1r8chP@ss"
+        }
+    },
+    "sqldbName": "sqlDbA",
+    "sqldbParameters": {
+        "location": "westus",
+        "properties": {
+            "collation": "SQL_Latin1_General_CP1_CI_AS"
+        }
+    }
+}
+  ```
+
+NOTE: For a list of collation values valid for use with Azure SQL Database, use this query:
+
+SELECT name, description
+FROM fn_helpCollations()
+
+3. Check the operation status of creating the service instance
+
+  The creating operation is asynchronous. You can get the operation status after the creating operation.
+
+  ```
+  cf service $service_instance_name
+  ```
+
+  For example:
+
+  ```
+  cf service mysqldb
+  ```
+
+[More information](http://docs.cloudfoundry.org/devguide/services/managing-services.html#create).
+
+## Using the services in your application
+
+### Binding
+
+  ```
+  cf bind-service $app_name $service_instance_name
+  ```
+
+  For example:
+
+  ```
+  cf bind-service demoapp mysqldb
+  ```
+
+### Format of Credentials
+
+  Verify that the credentials are set as environment variables
+
+  ```
+  cf env $app_name
+  ```
+
+  The credentials have the following format:
+  
+  ```
+  "credentials": {
+    "administratorLogin": "ulrich",
+    "administratorLoginPassword": "u1r8chP@ss",
+    "sqlServerName": "sqlServerA",
+    "sqldbName": "sqlDbA"
+  }
+
+  ```
```

Comment:
After the binding operation, I can't login to the SQL server because of the firewall rules.
```
"Cannot open server 'sqlservera' requested by the login. Client with IP address 'xxxxxx' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect.DB-Lib error message 40615, severity 14:\nGeneral SQL Server error: Check messages from the SQL Server\nDB-Lib error message 20002, severity 9:\nAdaptive Server connection failed\n" 
```
Can you setup the rules in the binding operation? If you can, a parameter about firewall rules should be added.